libr/core/cmd_anal.c

```diff
@@ -9260,9 +9272,12 @@ static int cmd_anal_all(RCore *core, const char *input) {
 				if (r_cons_is_breaked ()) {
 					goto jacuzzi;
 				}
-				oldstr = r_print_rowlog (core->print, "Check for objc references");
-				r_print_rowlog_done (core->print, oldstr);
-				cmd_anal_objc (core, input + 1, true);
+				const bool apple_target = is_apple_target (core);
+				if (apple_target) {
```

Comment:
```suggestion
				if (is_apple_target (core)) {
```
I would say this is short enough to avoid the use of another var